tests/test_interval.py

```diff
@@ -66,8 +67,8 @@ def test_interval_length() -> None:
     assert_type(i2 * 4.2, "pd.Interval[float]")
 
     if TYPE_CHECKING:
-        pd.Timestamp("2001-01-02") in i2  # type: ignore
-        i2 + pd.Timedelta(seconds=20)  # type: ignore
+        pd.Timestamp("2001-01-02") in i2  # pyright: ignore[reportGeneralTypeIssues]
+        i2 + pd.Timedelta(seconds=20)  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
This is one of these cases where `pyright` is correctly picking up the error, but `mypy` is not.